tests/python/test_matmul.py

```diff
@@ -201,3 +201,4 @@ def fusion_func(fd: FusionDefinition) -> None:
         ]
         outputs, _ = self.exec_nvfuser(fusion_func, inputs)
         assert outputs[0].ndim == 3
+        self.exec_nvfuser(fusion_func, inputs)
```

Comment:
what is this trying to do?

Reply:
This feels like leftover from a bad rebase.